cellpack/autopack/loaders/rename_dict.py

```diff
@@ -0,0 +1,21 @@
+v1_to_v2_name_map = {
```

Comment:
now that this file has more than just the rename dict, we should change the name of this file, to something like `v1-v2-attribute-changes`